dotcom-rendering/src/types/frontend.ts

```diff
@@ -126,6 +127,7 @@ export interface FEArticleType {
 	showTableOfContents: boolean;
 	lang?: string;
 	isRightToLeftLang?: boolean;
+	crossword?: CrosswordProps['data'];
```

Comment:
This is a larger point but I was surprised to see `crossword` as its own top-level property of the Frontend data model rather than as standard 'block'.

Something we can rationalise as it would mean we don't have to parse out the data separately in the enhancers and can treat as any other element type?